jax/lax_linalg.py

```diff
@@ -900,6 +914,9 @@ def svd_jvp_rule(primals, tangents, full_matrices, compute_uv):
   dU = jnp.matmul(U, F * (dSS + _T(dSS)))
   dV = jnp.matmul(V, F * (SdS + _T(SdS)))
 
+  if not compute_uv:
```

Comment:
Since you are returning early anyway, why not return as soon as you have `ds`?

Reply:
Addressed in last commit!